backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -130,17 +148,17 @@ public ResponseEntity<List<Object>> getUsers(
             }
             Optional<String> clientAndRolesJoins = Optional.empty();
             Optional<String> clientAndRolesCriteria = Optional.empty();
-            if (clientName.isPresent() && clientId.isPresent() && clientNameAndIdAreValid(clientName.get(), clientId.get())) {
+            if (clientClientId.isPresent() && clientId.isPresent() && clientIdAndIdAreValid(clientClientId.get(), clientId.get())) {
```

Comment:
In the future it would make sense to only pass a single identifier for the client (e.g. id) in the API call and then lookup the other values (e.g. clientId) via an API/db call. It also shouldn't be necessary to validate the values "match" in that case.